_sources/introduction/installation.rst

```diff
@@ -92,4 +92,8 @@ You can customize CLI-based installation (for example, specify directory, compil
 
      make -j VERBOSE=1 install
 
-
+Find More
+*********
+- `oneCCL Get Started Guide <https://www.intel.com/content/www/us/en/docs/oneccl/get-started-guide/2021-11/overview.html>`_
```

Comment:
not sure how it slipped through, I commented it at least twice and I saw a version that was ok; the link should lead to:
```suggestion
- `oneCCL Get Started Guide <https://www.intel.com/content/www/us/en/docs/oneccl/get-started-guide/current/overview.html>`_
```
right now, the link leads to outdated version of the website (to 2021.11, but the current release is 2021.12).